components/ide/code-desktop/status/main.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+
+	supervisor "github.com/gitpod-io/gitpod/supervisor/api"
+	"golang.org/x/xerrors"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Printf("Usage: %s <port> [<link label>] [<schema>]\n", os.Args[0])
+		os.Exit(1)
+	}
+	port := os.Args[1]
+
+	label := "Open in VS Code Desktop"
```

Comment:
`Open in VS Code Desktop Insiders` looks strange, maybe drop `Desktop`? 

or `Open in VS Code Insiders on Desktop`?

Reply:
Done